debezium-connector-mysql/src/main/java/io/debezium/connector/mysql/signal/KafkaSignalThread.java

```diff
@@ -135,7 +135,7 @@ private void monitorSignals() {
                     return;
                 }
                 catch (final Exception e) {
-                    LOGGER.error("Skipped signal due to an error '{}'", record, e);
+                    LOGGER.error("Skipped signal due to an error", e);
```

Comment:
For handling errors (here and below) - please add a new method to {{io.debezium.util.Throwables}}. The method will accept mutliple arguments

- logger
- error message
- record/data

The metod will log the error message at error level using passed logger and the there will be a {{TRACE}} level message like `Source of error is '{}'` using a new logger defined in `Throwables` class.

Reply:
Thanks for the suggestion. Makes sense.